install.go

```diff
@@ -354,9 +373,8 @@ func install(cmdArgs *settings.Arguments, dbExecutor db.Executor, ignoreProvider
 		_ = completion.Update(dbExecutor, config.AURURL, config.Runtime.CompletionPath, config.CompletionInterval, false)
 	}()
 
-	err = downloadPkgbuildsSources(do.Aur, incompatible)
-	if err != nil {
-		return err
+	if downloadOnly {
```

Comment:
seems that this will stop `downloadPkgbuildsSources` from running when it's in normal mode

Reply:
Yes, this for #1448.